packages/components/header/_autocomplete.scss

```diff
@@ -50,7 +50,11 @@
     width: 100%; /* [3] */
 
     &:focus {
-      box-shadow: inset 0 0 0 $nhsuk-box-shadow-spread $nhsuk-focus-color;
+      border: none;
```

Comment:
Duplicate border styles here (duplicated on line 54), which one is the correct one?

If its `border: none`, this needs to be `border: 0` instead.

Reply:
Removed `border: none;`